lib/_http_outgoing.js

```diff
@@ -368,7 +368,7 @@ OutgoingMessage.prototype._send = function _send(data, encoding, callback, byteL
     } else {
       const header = this._header;
       this.outputData.unshift({
-        data: header,
+        __proto__: null, data: header,
```

Comment:
```suggestion
        __proto__: null,
        data: header,
```

Reply:
fixed the rule to add multiline when needed